servers/rendering/renderer_rd/shader_rd.cpp

```diff
@@ -78,6 +80,29 @@ void ShaderRD::_add_stage(const char *p_code, StageType p_stage_type) {
 			chunk.type = StageTemplate::Chunk::TYPE_CODE;
 			push_chunk = true;
 			chunk.code = l.replace_first("#CODE", String()).replace(":", "").strip_edges().to_upper();
+		} else if (l.begins_with("#include \"")) {
```

Comment:
Note that this forces any custom shader code to only allow a single space between the keyword and the quotation symbol, which is different from both gdshader and GLSL

Reply:
Interestingly looking at the code in the python scripts, I wonder if they will work if a comment is added after the include.

Be that as it may, I've changed the code so it should be ok with any whitespace after `#include`